jjb/go-modules/go-mod-contracts.yaml

```diff
@@ -0,0 +1,17 @@
+---
+
+- project:
+    name: go-mod-contracts
```

Comment:
Would recommend all of the references to `go-mod-contracts` be changed to `go-mod-core-contracts` to match the repo name.